totest-manager.py

```diff
@@ -313,10 +313,10 @@ def maxsize_for_package(self, package):
         if re.match(r'.*-dvd5-.*', package):
             return 4700372992  # a DVD needs to match
 
-        if re.match(r'.*-image-livecd-x11.*', package):
+        if re.match(r'.*livecd-openSUSE:livecd-x11.*', package):
```

Comment:
Based on the final name below: `livecd-openSUSE:livecd-x11` it would seem the first `.*` is necessary and the second set since this is the package name, not binary.

Reply:
Could leave I suppose since this applies to other products where this may make sense even though I do not see any using it currently.